cmd/starter/main.go

```diff
@@ -29,10 +31,24 @@ func orFail(err error) {
 	}
 }
 
-func main() {
-	log = loggly.New(os.Getenv("LOGGLY_TOKEN"))
-	log.Tag("GoBuild-Starter")
+func init() {
+	log.Out = os.Stderr
 
+	papertrail_port, err := strconv.Atoi(os.Getenv("papertrail_port"))
+	if err != nil {
+		log.Panic("Failed to read papertrail_port")
+		os.Exit(1)
+	}
+	hook, err := logrus_papertrail.NewPapertrailHook(os.Getenv("papertrail_host"), papertrail_port, "GoBuilder Starter")
+	if err != nil {
+		log.Panic("Unable to create papertrail connection")
+		os.Exit(1)
+	}
+
+	log.Hooks.Add(hook)
```

Comment:
may want to allow starting up with a stderr logger instead of papertrail-- for trying it out, etc.